src/uu/mv/src/mv.rs

```diff
@@ -480,7 +494,8 @@ fn rename(
             if is_empty_dir(to) {
                 fs::remove_dir(to)?;
             } else {
-                return Err(io::Error::new(io::ErrorKind::Other, "Directory not empty"));
+                // TODO: change when 'io::Error::DirectoryNotEmpty' is stabilized.
```

Comment:
could you please add a pointer to get information about its stabilization ? thanks

Reply:
For sure. I'll add it as soon as Ifix the windows build on my fork